tests/antithesis/test-template/Dockerfile.client

```diff
@@ -1,35 +1,36 @@
-FROM ubuntu:latest
-
-# Install dependencies
-RUN apt-get update -y && apt-get install -y pip wget
-
-# GO:
-
-# Install go
-RUN wget https://golang.org/dl/go1.22.5.linux-amd64.tar.gz
-RUN tar -C /usr/local -xzf go1.22.5.linux-amd64.tar.gz
-ENV PATH=$PATH:/usr/local/go/bin 
+FROM golang:1.24 AS build
 
 # Set working directory
-WORKDIR /opt/antithesis/entrypoint
+WORKDIR /build
+
+# Copy replace directives for build
+# TODO - there should be a way to automate this
+COPY ./api ./api
+COPY ./client/pkg ./client/pkg
+COPY ./client/internal/v2 ./client/internal/v2 
+COPY ./client/v3 ./client/v3
+COPY ./etcdctl ./etcdctl
+COPY ./etcdutl ./etcdutl
+COPY ./pkg ./pkg
+COPY ./server ./server
+COPY ./tests ./tests 
```

Comment:
```suggestion
COPY . .
```

Reply:
Downside is that etcd root directory might include large file like binaries, etcd data directories from running it locally, etc. We might handle it later via dockerignore.

Reply:
Done, incorporated the suggestion :)